src/tests/examples/pytest.ini

```diff
@@ -0,0 +1,6 @@
+# content of pytest.ini
```

Comment:
Not sure to understand the role of this file...

Reply:
It is required by pytest to parse arguments